pkgs/data/themes/breath/default.nix

```diff
@@ -0,0 +1,47 @@
+{ mkDerivation
+, lib
+, fetchFromGitLab
+, gitUpdater
+, cmake
+, extra-cmake-modules
+, kdecoration
+, plasma-workspace
+, qtbase
+}:
+
+mkDerivation rec {
+  pname = "breath-theme";
+  version = "3130eb1addeebb4b93f1192c377383618ebecaa3";
```

Comment:
Please follow the [guidelines](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) on versioning. It should be `unstable-YYYY-MM-DD`.